### DISCUSSION TITLE
Question about Split Phase 220v

### DISCUSSION THREAD

**Discussion by moriahmorgan** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#35] [Q&A]
Hello,

I didn't see this explicitly spelled out in the Readme.MD / Wiki:

Do I need two clamps to measure a US based 220v circuit? Is there logic in ESPhome to deal with that? I seem to be getting mixed answers on this. If I need two clamps, do I need to do the severing operation to measure two voltages?

**Comment by presslab-us** [comment]
If your 240V circuit is single phase (no current on neutral) you can use one sensor.  If it's a split phase circuit (with a neutral) then you will need two sensors.

Some devices (like dryers) can look like single phase but are really split phase.  On older homes the ground connection is used as the neutral.

I believe with ESPHome you can use a multiply filter to multiply the power results by 2 to get the correct result with one sensor.
https://esphome.io/components/sensor/index.html#offset-multiply

Usually the voltages in split phase are close enough that you don't need two voltage measurements.  That's how IoTaWatt works by default.

**Comment by CircuitSetup** [comment]
> One more thing: You cannot measure the current flowing to a 240V load (dryer, stovetop, over, A/C, etc) with one current transformer (CT) clamped around both hot wires. You'll read zero because the two phase currents cancel each other magnetically. You do need two clamps (properly called CTs), one for each phase for correct measurements. However, since the voltage is the same on each phase (more or less) then the current flowing in each phase should be the same (if it's not, you have a problem) so it seems to me that measuring one phase and doubling the value would work just fine.

This is true, but if there is enough wire available, and the CT is large enough, you can pass both wires through 1 CT in opposite directions. Do not attempt this if the wire is a heavy gauge. Also, in most cases the load is balanced between both phases from the appliance, but some run electronics on 1 phase only. This likely wouldn't be an issue though since these electronics only use mA at a time.

I [updated the readme to include a section on this.](https://github.com/CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#measuring-dual-pole-240v-circuits)

**Comment by presslab-us** [comment]
A couple observations from my installation, regarding 240V appliances:

My dryer pulls about 150 watts from one phase only for the drum motor.  Because my home is older this current flows through the ground wire.  I needed to measure the current from both wires for this appliance.

My range, although it does have a dedicated neutral, only draw a few watts from this for the electronics so I ignored it and only measure one wire.

Because I couldn't route the two conductors through one cable clamp, I used an audio "Y" cable to combine two current sensors into one input.  As with two conductors in one clamp, care must be taken to ensure the measurement range of the input is not exceeded.
https://www.monoprice.com/product?p_id=30895
